security-admin/src/main/java/org/apache/ranger/rest/ServiceREST.java

```diff
@@ -2892,6 +2892,22 @@ private RangerPolicyEngine getPolicyEngine(String serviceName) throws Exception
 	public String checkSSO() {
 		return String.valueOf(bizUtil.isSSOEnabled());
 	}
+
+	@GET
+	@Path("/checkssotype")
+	@Produces({ "application/json"})
```

Comment:
:) space. I thought I already trained you to see spaces :)

Reply:
:) you are right. It looks like I didn't pay attention this after copy-paste :( will fix this.